doc/REST-interface.md

```diff
@@ -84,6 +84,13 @@ Returns various state info regarding block chain processing.
 Only supports JSON as output format.
 Refer to the `getblockchaininfo` RPC help for details.
 
+#### Deployment info
+`GET /rest/deploymentinfo/<BLOCKHASH>.json`
+Returns an object containing various state info regarding deployments of
+consensus changes at the current chain tip, or at <BLOCKHASH> if provided.
+Only supports JSON as output format.
```

Comment:
Is there a reason not to support the hex and bin formats, which most of the other endpoints support?